docs/NetCore_GettingStarted.md

```diff
@@ -70,3 +71,14 @@
     - `.\artifacts\bin\AnalyzerRunner\Release\netcoreapp3.1\AnalyzerRunner.exe ..\roslyn-analyzers\artifacts\bin\Microsoft.NetCore.Analyzers.Package\Debug\netstandard2.0 .\Roslyn.sln /stats /concurrent /a AnalyzerNameToTest /log Output.txt`
     - Do not forget change value after `/a` option with your testing analyzer name.
 The diagnostics reported by the analyzer will be listed in Output.txt.
+
+## Debugging analyzer with runtime repo projects
+
+1. Copy over debug build of analyzer assemblies on top of NetAnalyzers nuget package in your packages folder. (Instructions are same as the step 1 and 2 of [Testing against the Runtime repo step](#Testing-against-the-Runtime-and-Roslyn-analyzers-repo))
+2. Start VS and open a project you want to debug
+3. Note the process ID for `ServiceHub.RoslynCodeAnalysisService.exe` corresponding to that VS instance
```

Comment:
Do we want to add a section for debugging code fix provider as well? The code fix providers will execute in `devenv.exe`, so the step 3. here would be slightly different. If you do not want to duplicate the whole section, you can just add another line to this third step about code fixers.

Reply:
> Do we want to add a section for debugging code fix provider as well? The code fix providers will execute in devenv.exe, so the step 3. here would be slightly different. If you do not want to duplicate the whole section, you can just add another line to this third step about code fixers.

I would love to add that, but i don't know how it works, never tried

Reply:
Ah, you can just add a line to step 3 that if you want to debug the CodeFixProvider corresponding to the analyzer, then you will need to attach a different process, i.e. devenv.exe. Code fixes and analyzers run in different processes.